packages/docs/src/routes/(blog)/blog/components/mdx/article-hero/article-hero.tsx

```diff
@@ -0,0 +1,69 @@
+import { component$ } from '@builder.io/qwik';
+import { useDocumentHead } from '@builder.io/qwik-city';
+import { Image } from 'qwik-image';
+
+type Props = { imageSrc: string; authorLink: string };
+
+export const ArticleHero = component$<Props>(({ imageSrc, authorLink }) => {
+  const { title, frontmatter } = useDocumentHead();
+
+  if (!frontmatter.authorName || !frontmatter.tags || !frontmatter.date) {
+    return <>Missing frontmatter props</>;
+  }
+
+  return (
+    <>
+      <div class="pt-4">
+        <a
+          class="text-[color:var(--text-color)] flex items-center space-x-1 pl-2"
+          target="_blank"
+          rel="noopener"
+          href="/blog"
```

Comment:
should it open a new tab every time?

Reply:
Nice catch, let's remain in the same page